template/desktops/xfce.sh

```diff
@@ -19,6 +19,39 @@ xfconf-query -c xfce4-session -p /startup/ssh-agent/enabled -n -t bool -s false
 xfconf-query -c xfce4-session -p /startup/gpg-agent/enabled -n -t bool -s false
 echo "TIMING: $(date -Iseconds) - Disabled startup services"
 
+# Turn off power saving measures that turn off the display
+xfconf-query \
+    --channel xfce4-power-manager \
+    --property /xfce4-power-manager/dpms-enabled \
+    --create \
+    --type bool \
+    --set true
+
+xfconf-query \
+    --channel xfce4-power-manager \
+    --property /xfce4-power-manager/dpms-on-ac-off \
+    --create \
+    --set 0 \
+    --type uint
+
+xfconf-query \
+    --channel xfce4-power-manager \
+    --property /xfce4-power-manager/dpms-on-ac-sleep \
```

Comment:
Confirmed this property is `uint` type.

https://gitlab.xfce.org/xfce/xfce4-power-manager/-/blob/xfce4-power-manager-1.6.6/src/xfpm-xfconf.c#L421

```c
    /**
     * XfpmXfconf::dpms-on-ac-sleep
     **/
    g_object_class_install_property (object_class,
                                     PROP_DPMS_SLEEP_ON_AC,
                                     g_param_spec_uint (ON_AC_DPMS_SLEEP,
                                                        NULL, NULL,
							0,
							G_MAXUINT16,
							10,
                                                        G_PARAM_READWRITE));
```